reference/api/aut/index.md

```diff
@@ -291,7 +291,7 @@ Constraint checks:
 
 On successful processing of the method call:
 
-- the bonded Newton amount is locked in the `msg.Sender`'s Newton account
+- the bonded Newton amount is deducted from the `msg.Sender`'s Newton account balance and the corresponding amount of Newton is burned
```

Comment:
I think this is fine, but really the burning action is just a single operation in solidity (i.e. subtracting the value from the sender, and not adding it anywhere else).

Reply:
Change to:

>the bonded Newton amount is burned

Reply:
Done in commit e023406f88dc4d2c9188ed9616cecd8be614327a.